### DISCUSSION TITLE
Are you using react-svg-pan-zoom in your project?

### DISCUSSION THREAD

**Discussion by chrvadala** [chrvadala/react-svg-pan-zoom#190] [Show and tell]
Are you using **react-svg-pan-zoom** in your project? :grey_question: 

Share your project and show how you used **react-svg-pan-zoom** in your code. :technologist: :woman_technologist:

:star: :star: :star: :star: :star:

**Comment by cowchimp** [comment]
I created a VSCode extension called "[Source-Viz](https://marketplace.visualstudio.com/items?itemName=cowchimp.source-viz-vscode)" which is a tool for visualizing the relationship between a module's public API and its implementation details. If the matrix that's generated is larger than the viewport, it will show up wrapped in [an instance of react-svg-pan-zoom](https://github.com/cowchimp/source-viz/blob/046618a5faf4f6fcf9c91b533a0d8cc2f6b57b9e/packages/source-viz-core/src/components/matrix/ZoomWrapper.tsx#L24-L39) to make it easier to navigate. Thanks for creating this handy lib!

**Reply by chrvadala** [reply]
It's a perfect use case, thanks for sharing!

**Comment by RaghulR7** [comment]
Hi @chrvadala, 
We are using react-svg-pan-zoom for pan and zoom functions. Currently we are trying to implement a search functionality on our svg editor. For which we have to zoom into the specific position on the editor based on the search result. We are getting x and y coordinates and also the transform matrix from the search result object and we are trying to use the setPointOnViewerCenter method for zooming in to the position. But we are having trouble finding the exact position using the x,y and matrix. It will be of great help if you could gives us some inputs on how to resolve this or point us to any documentation regarding this. 
Thanks in Advance.

**Reply by chrvadala** [reply]
Thanks for sharing your experience with react-svg-pan-zoom. I answered to your question here https://github.com/chrvadala/react-svg-pan-zoom/discussions/194#discussioncomment-240365

**Comment by SmileHub** [comment]
Trying to use (in testing phase am atm)... but facing many incompletions yet in package. Written suggestions in discussion. There's a result so far:
![image](https://github.com/user-attachments/assets/4520f866-1c17-433c-94fa-7bf6d8f07c7f)
